librarian/core/collectors/commands.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import argparse
+
+from ..exports import ListCollector, to_list
+
+
+class Commands(ListCollector):
+    """
+    This class manages command argument handlers.
+    """
+    def __init__(self, supervisor):
+        super(Commands, self).__init__(supervisor)
+        self.parser = argparse.ArgumentParser()
+        self.handlers = {}
+
+    def collect(self, component):
+        commands = component.get_export('commands', [])
+        for command in commands:
+            try:
+                handler = component.get_object(command)
+            except ImportError:
+                logging.exception('Could not import handler')
+                continue
+            if not hasattr(handler, 'name'):
+                logging.error('Invalid handler {} for component {}'.format(
```

Comment:
Fixed by not doing this check at all, and instead letting the install_member() call raise AttributeError which is logged as a traceback.